packages/api/src/controllers/asset.ts

```diff
@@ -130,6 +130,7 @@ const fieldsMap: FieldsMap = {
 };
 
 app.get("/", authMiddleware({}), async (req, res) => {
+  res.header("Access-Control-Allow-Origin", "*");
```

Comment:
You should look into the cors plugin/library that we use somewhere in this repo. Should be in the places where we set up the whole express application like app-router or something like that

Reply:
Wouldn't adding the wildcard on the cors definition in `index.ts` add it to all the endpoints and not the VOD ones? I saw it was specifically added on some endpoints (e.g. the stream proxy)

Reply:
NVM, added only on vod via lib

Reply:
You need access-control-allow-method too

Reply:
added as well